docker/pypi/wmagent-base/install.sh

```diff
@@ -114,7 +114,7 @@ echo "-----------------------------------------------------------------------"
 stepMsg="Replace the current /data/manage script coming from 'dmwm-base' image with a symlink link"
 echo "-----------------------------------------------------------------------"
 echo "Start $stepMsg"
-[[ -f /data/manage ]] && rm -f /data/manage && ln -s $WMA_MANAGE_DIR/manage /data/manage
+[[ -f /data/manage ]] && ln -s $WMA_MANAGE_DIR/manage /data/manage
```

Comment:
Hi @amaltaro I know  you have not yet  asked for a review on this, but just noticed this change. This line would actually always fail. Let me put it with an example:
```
$ touch test.file
$ touch test.link
$ [[ -f test.link ]] && ln -s test.file test.link
ln: failed to create symbolic link 'test.link': File exists
``` 

Now let me also put the context of why the `&& rm -f /data/manage &&` was added at the first place. This file comes as the original `manage` script provided by the `dmwm-base` image [here](https://github.com/dmwm/CMSKubernetes/blob/11a4962c3dde96c639b62cbf5b3046bd06f3882c/docker/pypi/dmwm-base/Dockerfile#L18) and is completely different  from what we create  for the agent at `$WMA_MANAGE_DIR/manage`. So I decided to remove it an convert it from an executable file to a soft link pointing to the actual `manage` script that we are using with the agent. I could have left it untouched, but this would cause confusion as we would have two completely different manage scripts from two completely different sources in the same image, one of which is absolutely not related to the current component/service and impossible to work.